Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -3458,50 +3459,185 @@ def panorama_delete_url_filter_command(url_filter_name: str):
 ''' Security Rules Managing '''
 
 
+def xml_get(xml_dict: dict | Any, key: Any, subkeys=('member', '#text', '@name')) -> str | list | None:
+    """
+    Takes a dictionary converted from XML and a key and recursively "digs" through all the nested dicts,
+    using the subkeys only if needed, until it finds a non-dict value.
+    If it hits a list it will return a list of recursive calls of the list's elements.
+
+    For example:
+
+    In this case since one of the subkeys are in the sub dict, it will use it:
+    >>> xml_dict = {
+    ...     'key': {
+    ...         'random': 'foo',
+    ...         'subkey1': 'value'
+    ...     }
+    ... }
+    >>> xml_get(xml_dict, 'key', subkeys=('subkey1', 'subkey2'))
+    'value'
+
+    When none of the subkeys are in the sub dict it will pick a random key that does not begin with "@":
+    >>> xml_dict = {
+    ...     'key': {
+    ...         '@xml_attr': 'foo',
+    ...         'random': 'value'
+    ...     }
+    ... }
+    >>> xml_get(xml_dict, 'key', subkeys=('subkey1', 'subkey2'))
+    'value'
+
+    When a list is found it will return a list of recursive calls of the list's elements:
+    >>> xml_dict = {
+    ...     'key': [
+    ...         {
+    ...             'random': 'foo',
+    ...             'subkey1': 'value1'
+    ...         },
+    ...         {
+    ...             '@xml_attr': 'foo',
+    ...             'random': 'value2'
+    ...         }
+    ...     ]
+    ... }
+    >>> xml_get(xml_dict, 'key', subkeys=('subkey1', 'subkey2'))
+    ['value1', 'value2']
+
+    Args:
+        xml_dict (dict | Any): _description_
+        key (Any): _description_
+        subkeys (tuple, optional): _description_. Defaults to ('member', '#text', '@name').
+
+    Returns:
+        _type_ (str, list, None): _description_
+    """
+
+    if type(xml_dict) is not dict:
```

Comment:
```suggestion
    if not isinstance(xml_dict, dict):
```

please use `isinstance` instead of `is` in all places